instrumentation/opentelemetry-instrumentation-psycopg/src/opentelemetry/instrumentation/psycopg/__init__.py

```diff
@@ -237,15 +238,24 @@ def _uninstrument(self, **kwargs):
 
     # TODO(owais): check if core dbapi can do this for all dbapi implementations e.g, pymysql and mysql
     @staticmethod
-    def instrument_connection(connection, tracer_provider=None):
-        """Enable instrumentation in a psycopg connection.
+    def instrument_connection(
+        connection: psycopg.Connection,
+        tracer_provider: typing.Optional[trace_api.TracerProvider] = None,
+        enable_commenter: bool = False,
+        commenter_options: dict = None,
```

Comment:
Do we want `enable_attribute_commenter` as well?

Reply:
Yes! Added in [3587846](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3071/commits/358784640de4ee71aae99a84ad117a6fc10acf4f)